src/main/java/org/testng/TestRunner.java

```diff
@@ -514,16 +507,10 @@ private void initMethods() {
             m_excludedMethods,
             comparator);
 
-    m_allTestMethods =
-        MethodHelper.collectAndOrderMethods(
-            testMethods,
-            true /* forTest? */,
-            m_runInfo,
-            m_annotationFinder,
-            false /* unique */,
-            m_excludedMethods,
-            comparator);
-    m_classMethodMap = new ClassMethodMap(Arrays.asList(m_allTestMethods), m_xmlMethodSelector);
+    ITestNGMethod[] allTestMethods = computeAndGetAllTestMethods();
```

Comment:
Use `getAllTestMethods()` instead of `computeAndGetAllTestMethods`

Reply:
fixed.